src/backend/src/common/enums/pia-complexity.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum PiaComplexity {
+  LOW = 'LOW',
+  HIGH = 'HIGH',
```

Comment:
I think we also have medium complexity. Medium would most likely be a delegated, non-complex PIA.

Reply:
You're right, thanks! I had left a question to Peggy, which she responded that they call it "Standard". Updated the PR. ✅ Thanks.